master/buildbot/db/enginestrategy.py

```diff
@@ -113,19 +113,18 @@ def special_case_sqlite(self, u, kwargs):
 
     def set_up_sqlite_engine(self, u, engine):
         """Special setup for sqlite engines"""
+        import sqlite3
+        if sqlite3.sqlite_version_info < (3, 6, 19):
+            raise RuntimeError("Please use sqlite version > 3.6.19")
```

Comment:
Se suggest 3.7.0 in requirements.rst, but indicate that earlier versions are also supported.  So we should remove that indication from the docs.